### PR TITLE
Added #include to schema

### PR DESCRIPTION
**Pull request by GermanBluefox**

### CODE REVIEW

File: src/Components/SimpleCron/index.tsx [hunk 1]
```diff
@@ -739,10 +749,11 @@ class SimpleCron extends React.Component<SimpleCronProps, SimpleCronState> {
                             this.setState({ intervalBetween: _settings }, () => this.recalcCron());
                         }}
                     >
-                        {[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24].map(hour =>
+                        {[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23].map(hour =>
                             <MenuItem key={`A_${hour}`} value={hour}>
                                 {`${padding(hour)}:00`}
                             </MenuItem>)}
+                        {!settings.timeFrom && <MenuItem value={24}>24:00</MenuItem>}
```
Comment (foxriver76) [inline src/Components/SimpleCron/index.tsx:756]: what is the purpose of this, shouldn't it be 00:00?

Comment (GermanBluefox) [reply src/Components/SimpleCron/index.tsx:756]: From 23:00 to 00:00 looks strange

Comment (foxriver76) [reply src/Components/SimpleCron/index.tsx:756]: I mean okay for me but in my world 24:00 does not exist^

Comment (GermanBluefox) [reply src/Components/SimpleCron/index.tsx]: Ok. Replaced with 00:00

**Pull request merged**